addons/test_mail/tests/test_message_track.py

```diff
@@ -35,6 +35,84 @@ def test_message_track_author(self):
 
         self.assertEqual(self.record.message_ids.author_id, self.partner_admin)
 
+    def test_message_track_default_message(self):
+        def _track_get_message(self, field_names):
+            if 'customer_id' in field_names:
+                return 'Hello'
+            return super()._track_get_message(field_names)
+        self.patch(self.registry('mail.test.ticket'), '_track_get_message', _track_get_message)
+
+        with self.mock_mail_gateway():
+            self.record.customer_id = self.partner_admin
+            self.flush_tracking()
+
+        messages = self.record.message_ids
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages.body, '<p>Hello</p>', 'Default message should be used')
+
+        with self.mock_mail_gateway():
+            self.record._track_set_log_message('Hi')
+            self.record.customer_id = False
+            self.flush_tracking()
+
+        messages = self.record.message_ids - messages
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages.body, '<p>Hi</p>', '_track_set_log_message should take priority over default message')
+
+    def test_track_formatting(self):
+        def _track_filter_displayed_tracking_values(self, tracking_values):
```

Comment:
Same here, can't we paly with 'mail.test.track' whose method seems available for testing both behaviors (with and without filtering) ?